src/blr/rules/trim.smk

```diff
@@ -1,48 +1,70 @@
 """
 Rules connected to trimming of FASTQ files.
+
+READ1 LAYOUT
+
+5'-CAGTTGATCATCAGCAGGTAATCTGGBDVHBDVHBDVHBDVHBDVHCATGACCTCTTGGAACTGTCAGATGTGTATAAGAGACAGNNNN...NNNN(CTGTCTCTTATACACATCT)-3'
+   <------------h1----------><-------DBS--------><-----------------h2------------------><---gDNA--><---------h3-------->
+    h1 may inlude frameshift                                                                        Presence depencd on
+    oligos varying from 0-4                                                                         insert length
+    extra oligos in 5' end
+
+READ 2 LAYOUT
+
+5'-NNNN...NNNN(CTGTCTCTTATACACATCT)-3'
+   <---gDNA--><---------h3-------->
+              Presence depencd on
+              insert length
 """
 
+h1="CAGTTGATCATCAGCAGGTAATCTGG"
+DBS="N"*20
+h2="CATGACCTCTTGGAACTGTCAGATGTGTATAAGAGACAG"
+h3="CTGTCTCTTATACACATCT"
+trim_len=sum(map(len, [h1,DBS,h2]))
+extract_len=len(h1)
```

Comment:
Maybe these should be written to a config file? Since we've already changed a lot would could also postpone it to a a future PR/issue.